packages/client/src/exampleTypes/pivot.js

```diff
@@ -134,9 +134,9 @@ export default {
           ],
         })
       },
-      collapse(tree, type, drilldown) {
+      collapse(type, drilldown) {
```

Comment:
`expand` takes `tree` as the first argument. It's weird that `collapse` doesn't

Reply:
It's because mutate isn't exposed as an internal function. Let me see if I can address that, so they both don't need tree.